### PR TITLE
Bugfix/autocomplete small bugs

### PR DESCRIPTION
**Pull request by HybridFox**
## PR Checklist

This PR fulfills the following requirements:

- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)
- [x] A changelog entry has been added to CHANGELOG.md if necessary

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] react application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?
- Autocomplete does now not crash when a non-existing defaultValue is provided
- Sing Selection + Sync Mode now selects the correct value

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/).
 
 ## [Unreleased]
 
+### Fixed
+- Autocomplete does now not crash when a non-existing defaultValue is provided
+- Sing Selection + Sync Mode now selects the correct value
```
Comment (TriangleJuice) [inline CHANGELOG.md:11]: I'm not sure what this means.
```suggestion
- `Autocomplete` Fixed an issue where the correct value wasn't selected in async search mode
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/).
 
 ## [Unreleased]
 
+### Fixed
+- Autocomplete does now not crash when a non-existing defaultValue is provided
```
Comment (TriangleJuice) [inline CHANGELOG.md:10]:
```suggestion
- `Autocomplete` Fixed a crash when a non-existing defaultValue is provided
```

**Review by TriangleJuice — CHANGES_REQUESTED** [review]

**Review by TriangleJuice — APPROVED** [review]

**Comment by MichaelCastiau** [comment]
@HybridFox  Could you provide unit tests for the fix you made?

**Comment by MichaelCastiau** [comment]
@HybridFox  maybe best also to wait for the merge of https://github.com/digipolisantwerp/acpaas-ui_react/pull/169 since it contains an additional unit test file you may need.

**Comment by HybridFox** [comment]
> @HybridFox maybe best also to wait for the merge of #169 since it contains an additional unit test file you may need.

Wouldn't it make more sense to merge in the order of time opened :)
I will added the unit tests right now

**Review by MichaelCastiau — APPROVED** [review]

**Pull request merged**